searchform.php

```diff
@@ -1,8 +1,8 @@
 <form id="search" class="" method="get" action="<?php echo esc_url( home_url( '/' ) ); ?>"> 
   <div class="input-append">
 		<span aria-hidden="true" data-icon="&#xf002;"></span>
-		<input class="input-xlarge" type="text" name="s" value="<?php echo trim( get_search_query() ); ?>">
+		<input label="Search" class="input-xlarge" type="text" name="s" value="<?php echo trim( get_search_query() ); ?>">
 		<input type='hidden' name='post_type' value='programs,segments,staff,events,ads' />
-		<button id="searchsubmit" value="Search" type="submit" class="btn">Search</button>
+		<button label="button" id="searchsubmit" value="Search" type="submit" class="btn">Search</button>
```

Comment:
Label is not a valid attribute for the button element. Not sure what is being attempted here.

Reply:
Removed this label.